doc/Requirements Specification/Parts/Application.lyx

```diff
@@ -104,7 +104,49 @@ name "/A10/"
 
 \end_inset
 
- Re-engineering of existing software.
+ Beagle can be used for re-engineering of existing software.
+ When starting to use 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+Gls{Palladio}
+\end_layout
+
+\end_inset
+
+ for a existing software Beagle is helpful to detect bottlenecks and problems
```

Comment:
Yes, sorry.
I thougth you were referencing to line 107, and there should be no a/an at all.